node/src/components/storage.rs

```diff
@@ -1895,16 +1891,14 @@ impl Storage {
         txn: &mut Tx,
         block_header: &BlockHeader,
     ) -> Result<Option<BlockSignatures>, FatalStorageError> {
-        if let Some(last_emergency_restart) = self.last_emergency_restart {
-            if block_header.era_id() <= last_emergency_restart {
-                warn!(
-                    ?block_header,
-                    ?last_emergency_restart,
-                    "finality signatures from before last emergency restart requested"
-                );
-                return Ok(None);
-            }
+        if block_header.is_first_after_emergency_restart() {
+            warn!(
```

Comment:
Is that still a reason to `warn!`? We do intend to sign them.

Reply:
This `if` shouldn't even be there, as there should be no `is_first_after_emergency_restart` now (as you noted below). I'll just remove it.